subprojects/packagefiles/glfw/tests/meson.build

```diff
@@ -0,0 +1,51 @@
+###  GLFW test programs
+#@ Build check
+if not get_option('build-tests')
+  subdir_done()
+endif
+
+#@ Constants
+tst_include = include_directories('../include', '../src', '../deps')
+glad_gl = '../deps/glad_gl.c'
+glad_vulkan = '../deps/glad_vulkan.c'
+getopt = '../deps/getopt.c'
+tinycthread = '../deps/tinycthread.c'
+
+#@ Flags
+if is_posix == true
+  c_flags = ['-D_XOPEN_SOURCE=700']
+endif
+
+#@ Test executables
+tests = [['clipboard', [getopt, glad_gl], ['console', 'glfw']],
+         ['events', [getopt, glad_gl], ['console', 'glfw']],
```

Comment:
Every single test seems to be tagged with the "glfw" suite, which is redundant because Meson includes a builtin scope for the subproject itself, hence `--suite glfw` will already select that. :)